plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestCheckpointWriter.java

```diff
@@ -78,8 +78,11 @@
 @Test
 public class TestCheckpointWriter
 {
+    private static final ConnectorSession SESSION = TestingConnectorSession.builder()
+            .setPropertyMetadata(new DeltaLakeSessionProperties(new DeltaLakeConfig(), new ParquetReaderConfig(), new ParquetWriterConfig()).getSessionProperties())
```

Comment:
Can we do this once, not in every test class?

Reply:
I can extract to some helper class.